quix-frontend/client/src/lib/sql-autocomplete/adapter/tests/get-completion-tests/expected-results.ts

```diff
@@ -34,118 +34,132 @@ export const expectedResult = {
+    makeCompletionItem('tblName1.col2_of_tblRef2', 'varchar'),
+    makeCompletionItem('col2_of_tblRef2', 'varchar'),
+    makeCompletionItem('col1', 'column'),
     makeCompletionItem('tblName1.col1', 'column'),
+    makeCompletionItem('col2', 'column'),
     makeCompletionItem('tblName1.col2', 'column'),
-    makeCompletionItem('ctl1.schm1.extTbl1.col1_of_extTbl1', 'column'),
-    makeCompletionItem('ctl1.schm1.extTbl1.col2_of_extTbl1', 'column'),
+    makeCompletionItem('ctl1.schm1.extTbl1.col1_of_extTbl1', 'varchar'),
+    makeCompletionItem('col1_of_extTbl1', 'varchar'),
+    makeCompletionItem('ctl1.schm1.extTbl1.col2_of_extTbl1', 'varchar'),
+    makeCompletionItem('col2_of_extTbl1', 'varchar'),
   ],
   threeTbl1Wtwithcolumns12AndTblRef1Ext1ExtwithAlias: [
     makeCompletionItem('col1', 'column'),
-    makeCompletionItem('col2', 'column'),
-    makeCompletionItem('col1_of_tblRef1', 'column'),
-    makeCompletionItem('col2_of_tblRef1', 'column'),
-    makeCompletionItem('col1_of_extTbl1', 'column'),
-    makeCompletionItem('col2_of_extTbl1', 'column'),
     makeCompletionItem('tblName2.col1', 'column'),
+    makeCompletionItem('col2', 'column'),
     makeCompletionItem('tblName2.col2', 'column'),
-    makeCompletionItem('tblName2.col1_of_tblRef1', 'column'),
-    makeCompletionItem('tblName2.col2_of_tblRef1', 'column'),
-    makeCompletionItem('ctl1.schm1.extTbl1.col1_of_extTbl1', 'column'),
-    makeCompletionItem('ctl1.schm1.extTbl1.col2_of_extTbl1', 'column'),
-    makeCompletionItem('tblAlias1.col1_of_extTbl1', 'column'),
-    makeCompletionItem('tblAlias1.col2_of_extTbl1', 'column'),
-  ],
+    makeCompletionItem('tblName2.col1_of_tblRef1', 'varchar'),
+    makeCompletionItem('col1_of_tblRef1', 'varchar'),
+    makeCompletionItem('tblName2.col2_of_tblRef1', 'varchar'),
+    makeCompletionItem('col2_of_tblRef1', 'varchar'),
+    makeCompletionItem('ctl1.schm1.extTbl1.col1_of_extTbl1', 'varchar'),
+    makeCompletionItem('col1_of_extTbl1', 'varchar'),
+    makeCompletionItem('ctl1.schm1.extTbl1.col2_of_extTbl1', 'varchar'),
+    makeCompletionItem('col2_of_extTbl1', 'varchar'),
+    makeCompletionItem('tblAlias1.col1_of_extTbl1', 'varchar'),
+    makeCompletionItem('tblAlias1.col2_of_extTbl1', 'varchar'),
+  ],
+  expected18 : [
+    { value: 'catalog0.schema0.TV.movies' , meta: 'row' , caption: 'movies' },
+    { value: 'catalog0.schema0.TV.tvSeries' , meta: 'row' , caption: 'tvSeries' },
+  ],
+  expected19 : [
+    { value: 'catalog0.schema0.TV.movies.action' , meta: 'row' , caption: 'action' },
+    { value: 'catalog0.schema0.TV.movies.comedy' , meta: 'row' , caption: 'comedy' },
+  ],
+  expected20 : [
+    { value: 'catalog0.schema0.TV.movies.action.The_Dark_Knight' , meta: 'varchar' , caption: 'The_Dark_Knight' },
+    { value: 'catalog0.schema0.TV.movies.action.The_Avengers' , meta: 'varchar' , caption: 'The_Avengers' },
+    { value: 'catalog0.schema0.TV.movies.action.Die_Hard' , meta: 'varchar' , caption: 'Die_Hard' },
+  ],
```

Comment:
expected18? 19? 20? 21? make it readable

we need to watch die hard as a team.